src/config/config.js

```diff
@@ -1,28 +1,28 @@
-/*
-    To access any of the configuration settings in this file, use:
-        import config from "[path to config.js]"
-    Then you can access the settings as "config.apiUrl", "config.streamingUrl", etc
-*/
-
-// var config = {
-//     apiUrl: "http://172.19.145.91/api/",
-//     streamingUrl: "http://172.19.145.91/"
-// };
-//
-//  var config = {
-//      apiUrl: "http://127.0.0.1:8000/api/",
-//      streamingUrl: "http://127.0.0.1/"
-// };
-
-
-const config = {                                                        //remote api
-    apiUrl: "https://localhost/api/",
-    streamingUrl: "https://localhost/"
-};
-
-// const config = {                                                        //remote api
-//     apiUrl: "https://te.loc/api/",
-//     streamingUrl: "https://te.loc/"
-// };
-
-export default config;
+/*
+    To access any of the configuration settings in this file, use:
+        import config from "[path to config.js]"
+    Then you can access the settings as "config.apiUrl", "config.streamingUrl", etc
+*/
+
+// var config = {
+//     apiUrl: "http://172.19.145.91/api/",
+//     streamingUrl: "http://172.19.145.91/"
+// };
+//
+// var config = {
+      //apiUrl: "http://127.0.0.1:8000/api/",
+      //streamingUrl: "http://127.0.0.1/"
+ //};
+
+
+const config = {                                                        //remote api
+	apiUrl: "http://localhost:8000/api/",

```

Comment:
hey so i just started looking through this PR, was just curious as to the reason you had the port number? Is it necessary for the tests to run?

Reply:
I added the port number for clarity's sake. I assumed that without the port number, the front end would be looking in the wrong port for the API. I did not try running the tests with out the port number. If it still works and there's a preference, then I can change it back.